File: StudentCredit.Application/Applications/ApplicationsFive/Dtos/ApplicationFiveSearchResultItemDto.cs
using StudentCredit.Data.Applications.ApplicationFive.Enums;
using StudentCredit.Data.Common.Enums;
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Applications.ApplicationsFive.Dtos
{
	public class ApplicationFiveSearchResultItemDto
	{
		public int Id { get; set; }

		public DateTime BlankDate { get; set; }

		public DateTime? From { get; set; }

		public DateTime? To { get; set; }

		public NomenclatureDto Bank { get; set; }

		public ApplicationFiveType ApplicationFiveType { get; set; }

		public CommitState CommitState { get; set; }
	}
}

File: WebApplication/src/modules/application/application-five/models/application-five-search-result-item.dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { ApplicationFiveType } from "../enums/application-five-type.enum";
import { CommitState } from "../../common/enums/commit-state.enum";

export class ApplicationFiveSearchResultItemDto {
  id: number;
  blankDate: Date;
  from: Date;
  to: Date;
  bank: NomenclatureDto;
  applicationFiveType: ApplicationFiveType;
  commitState: CommitState;
}